src/views/postCreation/PostCreation.tsx

```diff
@@ -304,7 +304,7 @@ export const PostCreation: React.FC<IPostCreationProps> = ({
         title: savedPostDraft.title,
         videoUrl: savedPostDraft.videoUrl,
         type: { id: postType.type, name: postType.name },
-      } as IPost),
+      } as unknown) as IPost),
```

Comment:
why double type assertion was needed here?

Reply:
![Screenshot 2022-10-13 at 22 29 40](https://user-images.githubusercontent.com/40029364/195692055-c9ae4c39-fc47-4390-8eed-daa5e6dd555a.png)

Reply:
ideally we might have `IPost` without `id` and `type ISavedPost = IPost & {  id: number }` but if that's a lot of changes, let's keep it as double conversion(but I would still consider this as a techdebt)

Reply:
...or in opposite: `IUnsavedPost = Omit<IPost, "id">`